spec.md

```diff
@@ -423,13 +423,17 @@ Manifest byte stream:
 
 The uploaded manifest MUST reference any blobs that make up the artifact.
 However, the list of blobs MAY be empty.
+
+The registry MUST store the manifest in the exact byte representation provided by the client.
```

Comment:
I might've worded "in the exact byte representation" more like "byte for byte exactly as"

Reply:
Since this one is merged, any wording changes now should be pushed as a new PR.

Reply:
:-)